File: mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
   int height;
   int c = 0;

   do{
       printf("Welche Hoehe?\n");
       height = get_int();
   }while(height < 0 || height >= 24);

   for(int rows = height; rows > 0; rows--)
   {
       for(int space = 0; space < rows - 1; space++)
       {
           printf(" ");
       }
       printf("#");
       c++;
       for (int sethashes = 0; sethashes < c - 1; sethashes++)
       {
           printf("#");
       }
       printf("  ");
       for (int sethashes = 0; sethashes < c; sethashes++)
       {
           printf("#");
       }
       printf("\n");
   }
   return 0;
}

File: sentimental/mario/mario.py
from cs50 import get_int

def main():
    while True:
        print("Height: ", end = "")
        height = get_int()
        if height >= 0 and height <= 23:
            break

    for rows in range(height):
        for space in range(height - rows - 1):
            print(" ", end = "")
        for hashes in range(height - rows, height + 1):
            print("#", end = "")
        print("  ", end = "")
        for hashes in range(height - rows, height + 1):
            print("#", end = "")
        print()

if __name__ == "__main__":
    main()

